### PR TITLE
add techforpalestine datasets

### PR DESCRIPTION
**Pull request by sterlingwes**
👋🏻  thanks for the great resource

TFP offers three JSON datasets via our website:
* list of those killed in gaza (14k names as of writing and we're improving the english name translations)
* daily reports on those killed, injured
* summary JSON of the latest values from the daily report dataset

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -273,6 +273,7 @@ Thanks. please refer to [this](https://github.com/aborazmeh/awesome-palestine/bl
 ## APIs
 
 - 🌐 [List of API endpoints providing data related to Palestine.](https://github.com/Zain-ul-din/awesome-palestine-apis)
+- 🌐 [data.techforpalestine.org](https://data.techforpalestine.org)
```
Comment (sentientist-tech) [inline README.md]:
```suggestion
- 🌐 [Palestine Datasets by TFP](https://data.techforpalestine.org)
```

Comment (sentientist-tech) [reply README.md]: just a suggestion. Always good to have a link text that describes the link rather than have the link as the text

**Comment by aborazmeh** [comment]
Thank you,
I was working on something similar, and I translated many of the names.
I will attach my work here, so you can use it if you found it useful.
[martyrs.json](https://github.com/aborazmeh/awesome-palestine/files/14181271/martyrs.json)

- Please note that these lists, despite being official (by the [Palestinian Ministry of Health](http://moh.gov.ps/)), are in no way complete. Even people who lost their lives in October 2023 are not in the lists, as their bodies are still unfound or under the rubble. Maybe we should add a note explaining that.

**Comment by sterlingwes** [comment]
Thank you @aborazmeh , yes we included the caveats from the ministry under Sources: https://data.techforpalestine.org/docs/killed-in-gaza/#source if you think that's not enough let me know. We'll work on improving how we describe the data though.

Thank you for your list! I'll compare and see if we can improve our translation setup. Can I assume that the ones you've translated are those with a `name` field without any arabic showing up in it?

**Comment by aborazmeh** [comment]
> Can I assume that the ones you've translated are those with a name field without any arabic showing up in it?

Not really. For the sake of consistency, I used regex to replace **whole** words, so you might find some names partly translated, as the authorities in Palestine uses the quadruple naming system for official documents.

**Pull request merged**